ui/src/app/cohort-review/page-layout/page-layout.ts

```diff
@@ -1,31 +1,50 @@
-import {Component, OnInit} from '@angular/core';
+import {Component, OnDestroy, OnInit} from '@angular/core';
 
 import {cohortReviewStore} from 'app/cohort-review/review-state.service';
-import {navigate, urlParamsStore} from 'app/utils/navigation';
-import {ReviewStatus} from 'generated';
+import {currentCohortStore, currentWorkspaceStore, navigate, urlParamsStore} from 'app/utils/navigation';
+import {CohortReviewService, CohortsService, PageFilterRequest, PageFilterType, ParticipantCohortStatusColumns, ReviewStatus, SortOrder} from 'generated';
 
 @Component({
   templateUrl: './page-layout.html',
   styleUrls: ['./page-layout.css']
 })
-export class PageLayout implements OnInit {
-
-  subscription: any;
-  create = false;
-  constructor() {}
+export class PageLayout implements OnInit, OnDestroy {
+  reviewPresent: boolean;
+  cohortLoaded = false;
+  constructor(
+    private reviewAPI: CohortReviewService,
+    private cohortsAPI: CohortsService
+  ) {}
 
   ngOnInit() {
-    const review = cohortReviewStore.getValue();
+    const {ns, wsid, cid} = urlParamsStore.getValue();
+    const cdrid = +(currentWorkspaceStore.getValue().cdrVersionId);
+    this.reviewAPI.getParticipantCohortStatuses(ns, wsid, cid, cdrid, {
+      page: 0,
+      pageSize: 25,
+      sortColumn: ParticipantCohortStatusColumns.PARTICIPANTID,
```

Comment:
Existing issue, but: `sortColumn` seem to be a real request field. Should be able to remove and then subsequently this should allow you to remove the below `PageFilterRequest` cast